view/frontend/web/js/express-pay-storefront.js

```diff
@@ -44,7 +45,7 @@ define([
             let boldPaymentsInstance;
 
             try {
-                boldPaymentsInstance = await spi.getPaymentsClient();
+                boldPaymentsInstance = await spi.getPaymentsClient(this.pageSource);
```

Comment:
Where does `this.pageSource` come from? There's no variable on the component to store this. I assume it's coming from somewhere external? Are we guaranteed to have it at this point

Reply:
It's passed in through the PHTML, in express-pay.phtml you can see the pageSource passed in through the script loaded in the HTML. This comes from the block in the view XML as an <argument />

Reply:
I believe it's good practice to put a variable on the Component so everything can match up. Otherwise this just looks like a mystery and has no default value if something breaks. You can add defaults if you see here: https://developer.adobe.com/commerce/frontend-core/ui-components/concepts/class/#commonly-used-properties